### DISCUSSION TITLE
How about that user can set tilting row and column as they want?

### DISCUSSION THREAD

**Discussion by ghreyt** [rxhanson/Rectangle#1450] [Ideas]
Hello, I have just installed Recangle Pro and really love it.

But just I hope one feature.
The feature that I want/think is this.

1. User can add customized tiling as they want
  - For example, user can add 4x2 tilging
2. And user can set shortcut to it 
3. Moreover, if possible, the added tiling also can be shown in menu with submenu automatically

I am not sure if this is what any other needs
but for me it is the feature that I really want to get.

Especially customized tiling with shortcut.
Sometimes I opens many windows at the same time (mostly Slack) and want to tile them
to catch new message as soon as possible. 😄

**Comment by rxhanson** [comment]
Thanks for the idea! This one is actually on my backlog; it's been requested a couple of times before. I don't have a timeframe yet on when I will get to it, though.
